### PR TITLE
controlplane/authz: Add source service to a connection request to the policy engine

### PR DESCRIPTION
**Pull request by kfirtoledo**
Add source service to a connection request to the policy engine.

### CODE REVIEW

**Review by praveingk — APPROVED** [review]

File: pkg/controlplane/authz.go [hunk 1]
```diff
@@ -79,12 +79,20 @@ func (cp *Instance) AuthorizeEgress(req *EgressAuthorizationRequest) (*EgressAut
 		return nil, fmt.Errorf("no bindings found for import '%s'", req.Import)
 	}
 
-	// TODO: get k8s attributes using cp.kubeClient
 	connReq := eventmanager.ConnectionRequestAttr{DstService: req.Import, Direction: eventmanager.Outgoing}
+	srcLabels := cp.platform.GetLabelsFromIP(req.IP)
+
+	if srcLabels != nil {
```
Comment (zivnevo) [inline pkg/controlplane/authz.go:85]: no need to check for nil value. [The nil map supports queries](https://go.dev/ref/spec#Index_expressions), and will return the same results as an empty map.
It's only insertions to the nil map that panic.

**Review by zivnevo — APPROVED** [review]
Just these two nits

File: pkg/controlplane/authz.go [hunk 2]
```diff
@@ -79,12 +79,20 @@ func (cp *Instance) AuthorizeEgress(req *EgressAuthorizationRequest) (*EgressAut
...
+		if src, ok := srcLabels["app"]; ok {
+			connReq.SrcService = src
+		}
+	}
```
[the remark this answers is not in the document]
Comment (kfirtoledo) [reply pkg/controlplane/authz.go]: Done

File: pkg/controlplane/authz.go (hunk 1 above)
Comment (kfirtoledo) [reply pkg/controlplane/authz.go]: Done

**Pull request merged**